### PR TITLE
update passportforwork-csp.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #2488 
added info about GUID

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/passportforwork-csp.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The following diagram shows the PassportForWork configuration service provider i
 Root node for PassportForWork configuration service provider.
 
 <a href="" id="tenantid"></a>***TenantId***  
-A globally unique identifier (GUID), without curly braces ( { , } ), that is used as part of Windows Hello for Business provisioning and management.
+A globally unique identifier (GUID), without curly braces ( { , } ), that is used as part of Windows Hello for Business provisioning and management. To get the GUID, use the [Get-AzureAccount](https://docs.microsoft.com/powershell/module/servicemanagement/azure/get-azureaccount) PowerShell cmdlet. For more information please see [Get Windows Azure Active Directory Tenant ID in Windows PowerShell](https://devblogs.microsoft.com/scripting/get-windows-azure-active-directory-tenant-id-in-windows-powershell).
```
Comment (nenonix) [inline windows/client-management/mdm/passportforwork-csp.md]:
```suggestion
A globally unique identifier (GUID), without curly braces ( { , } ), that is used as part of Windows Hello for Business provisioning and management. To get the GUID, use the  PowerShell cmdlet [Get-AzureAccount](https://docs.microsoft.com/powershell/module/servicemanagement/azure/get-azureaccount). For more information see [Get Windows Azure Active Directory Tenant ID in Windows PowerShell](https://devblogs.microsoft.com/scripting/get-windows-azure-active-directory-tenant-id-in-windows-powershell).
```

**Comment by MaratMussabekov** [comment]
Dear @nenonix , your recommendations were applied, thank you!

**Comment by mypil** [comment]
@AndreaBarr @lizap @jaimeo @Justinha @DaniHalfin @mjcaparas @greg-lindsay - Can anyone of you please assist to check, approve and merge the content of this PR based on issue #2488?

Thank you.

**Pull request merged**